teachertool/src/services/backendRequests.ts

```diff
@@ -2,6 +2,21 @@ import { stateAndDispatch } from "../state";
 import { ErrorCode } from "../types/errorCode";
 import { logError } from "./loggingService";
 
+export async function fetchJsonDocAsync<T = any>(url: string): Promise<T | undefined> {
+    try {
+        // TODO: Prepend CDN origin if not localhost
```

Comment:
So far in beta or uploaded targets, I don't think we've seen any CDN-related issues for the doc files w/ the catalog and validator plans (though I could be missing something). Are you anticipating issues specific to prod?